src/layers/legacy/medCoreLegacy/gui/factories/medToolBoxFactory.h

```diff
@@ -69,9 +69,16 @@ class MEDCORELEGACY_EXPORT medToolBoxFactory : public dtkAbstractFactory
                                toolboxType::staticName(),
                                toolboxType::staticDescription(),
                                toolboxType::staticCategories(),
-                               creator);
+                               creator,
+                               nullptr);
     }
 
```

Comment:
The non-templated registerToolBox function below is moved to the public section to be accessible from Python.